docs/architecture/adr/ADR043-k8s-resource-access.md

```diff
@@ -0,0 +1,530 @@
+# ADR043: Kubernetes resource access rules
+
+## Status
+
+Pending
+
+## Context
+
+Several different levels of access are required within a kubernetes cluster. The
+GSP uses role based access control so these levels are granted to users and
+groups via roles.
+
+## Decision
+
+We will create two levels of access within each namespace:
+
+* Operator
+* Auditor
+
+The Operator is a relatively permissive read-write role within the namespace.
+Developers working on branches that are not part of the release process may be
+granted this role in certain namespaces. This is also the role the in-cluster
+concourse team for each namespace will be granted.
+
+The Auditor is mostly read-only and will be given to all authenticated users in
```

Comment:
`Auditor:   delete` shows up 60 times in the document.

Reply:
Your point being that we should remove the "mostly read-only" from the description?

Reply:
probably

Reply:
I've made some changes to the language around this role. Could you take another quick look?

https://github.com/alphagov/gsp/pull/842/commits/36969602abb6121750188530bd123ab19169ed32

Reply:
sounds good